### PR TITLE
Js2/week1/bani

### PR DESCRIPTION
**Pull request by banirezaie**

### CODE REVIEW

File: week-1/Homework/mandatory/5-magic-8-ball.js [hunk 1]
```diff
@@ -55,8 +100,52 @@ let answer;
...
+  } else if (number === 16) {
+    return "verr negative";
+  } else if (number === 17) {
+    return "verr negative";
+  } else if (number === 18) {
+    return "verr negative";
+  } else if (number === 19) {
+    return "verr negative";
+  } else if (number === 20) {
+    return "verr negative";
```
Comment (DomVinyard) [inline week-1/Homework/mandatory/5-magic-8-ball.js:143]: There's a quicker (and in my opinion more fun) way to do this! 

```
if (number < 6) {
  return "verr positive"
}
else if (number < 11) {
  return "positive"
}
else if ...
```

Much less typing 😁

File: week-4/Homework/mandatory/2-writers.js [hunk 2]
```diff
@@ -38,6 +38,13 @@ let writers = [
     alive: true
   }
 ];
+writers
+  .filter(x => x.alive)
```
Comment (DomVinyard) [inline week-4/Homework/mandatory/2-writers.js:42]: 👏

File: week-4/Homework/mandatory/3-water-bottle.js [hunk 3]
```diff
@@ -12,12 +12,17 @@ let bottle = {
...
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    return (this.volume -= 10);
   },
   empty: function() {
     // this function should return true if your bottles volume = 0
+    if (this.volume == 0) {
+      return true;
+    }
```
Comment (DomVinyard) [inline week-4/Homework/mandatory/3-water-bottle.js:25]: you can also instead simply write

```
if (!this.volume) {
   return true
}
```

or if you want to go even further, we can remove the `if` entirely.

```
empty: function() {
  return !this.volume
}
```

Can you see how these three are equivalent?

File: week-4/InClass/A-objects-intro/exercise-part-1.js [hunk 4]
```diff
@@ -7,3 +7,43 @@ Assign each of them to a separate variable
...
+  model: "electric",
+  brand: "iRider",
+  madeIn: "china",
+  maxSpeed: 30
+};
+let book = {
+  name: "Bible",
+  pages: 500,
+  coverColour: "black",
+  language: "persian"
```
Comment (DomVinyard) [inline week-4/InClass/A-objects-intro/exercise-part-1.js:48]: I thought "Aramaic"?

File: week-4/InClass/E-arrays-of-objects/exercise-1.js [hunk 5]
```diff
@@ -5,42 +5,55 @@
...
 */
 
+let persons = [person1, person2, person3]; // Complete here
 
-var persons = // Complete here
-
-var personNames = // Complete here
+let personNames = persons.map(x => x.name); // Complete here
 
 var personsYoungerThan28YearsOld = // Complete here
```
Comment (DomVinyard) [inline week-4/InClass/E-arrays-of-objects/exercise-1.js:31]: The way you did `personNames` above is a good place to start here

File: week-4/InClass/A-objects-intro/exercise-part-1.js (hunk 4 above)
Comment (banirezaie) [reply week-4/InClass/A-objects-intro/exercise-part-1.js:48]: "Aramic" is a part of Persian language which was the main language of Iran in the `Sasanian Empire` Era.

**Pull request closed without merging**